highfidelity/hifi
# Make setBlendshape changes persist

Fix for: https://highfidelity.fogbugz.com/f/cases/2782/MyAvatar-setBlendshape-Resets-after-a-single-frame

Testing Instructions:
- Try commands such as:
     > MyAvatar.setBlendshape("MouthSmile_R", 1.0);
     > MyAvatar.setBlendshape("MouthSmile_L", 1.0);
- Verify that the changes persist beyond just one frame
